app/src/main/java/com/concordium/wallet/ui/identity/identityproviderwebview/IdentityProviderWebViewViewModel.kt

```diff
@@ -220,12 +220,12 @@ class IdentityProviderWebViewViewModel(application: Application) : AndroidViewMo
                     override fun onResponse(call: Call<String>, response: Response<String>) {
                         val response = response.errorBody()?.string()
                         when {
-                            response?.contains(CreateIdentityError.NONE.message) == true -> {
+                            CreateIdentityError.NONE.containsStatusMessage(response) -> {
```

Comment:
To me this reads a little weird. When you have:
```
CreateIdentityError.NONE.containsStatusMessage()
```
then it sounds like that you are checking if the `ENUM` value `NONE` contains some status message, but we are actually checking the `response`.

Reply:
changed to 
![image](https://github.com/Concordium/concordium-reference-wallet-android/assets/11137601/dd819794-d382-4d9b-8938-8d819d14b519)